src/Components/App/App.jsx

```diff
@@ -49,20 +50,30 @@ function App() {
     }
   };
 
+  const resetCustomTip = () => {
+    setCustomTip("");
```

Comment:
Nice! This makes a lot of sense to bring the customTip state up the App component and define a function (resetCustomTip) to functionally reset the state to an empty string.